CMakeLists.txt

```diff
@@ -243,7 +243,20 @@ ENDIF()
 
 OPTION(WITH_MSAN "Enable memory sanitizer" OFF)
 IF (WITH_MSAN)
+  MY_CHECK_AND_SET_COMPILER_FLAG("-fdebug-macro" DEBUG)
```

Comment:
This is something generic to help debug code that was compiled with `clang`. It is not strictly needed. You could for example see the values of some `#define` constants in `gdb`.